### PR TITLE
add port 9003 to the document

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

File: docs/minio_guide.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ interacting with MinIO.
 
     ```bash
     ssh -f -N -L 9002:ci07:9002 <ac.anl_username>@login1.berkeley.kbase.us 
+    ssh -f -N -L 9003:ci07:9003 <ac.anl_username>@login1.berkeley.kbase.us
```
Comment (MrCreosote) [inline docs/minio_guide.md:13]: If the plan is to give out access keys that can't log into the UI, is this still necessary?

Comment (Tianhao-Gu) [reply docs/minio_guide.md:13]: yea. I guess just in case they need to. It won't hurt I guess.

Comment (MrCreosote) [reply docs/minio_guide.md:13]: Ok, maybe make a note that the UI isn't available to ordinary access creds so people aren't confused when they can't log in?

Comment (Tianhao-Gu) [reply docs/minio_guide.md:13]: 👍

**Review by MrCreosote — APPROVED** [review]

**Pull request merged**